backend/app/api_cheese.py

```
@@ -667,30 +667,101 @@ def send_state(session, app, ap_position, is_tracked, current_user_id_for_thread
+                    if local_slot:
+                        logging.info(f"[CHEESE_DEBUG] Collision: Untracking slot {ap_position} locally.")
+                        session.delete(local_slot)
+                        
+                        player_name = f"Slot {ap_position}"
+                        try:
+                            players = json.loads(room.cached_players_json or '[]')
+                            for p in players:
+                                if p.get('slot_id') == ap_position:
+                                    player_name = p.get('alias') or p.get('name') or player_name
+                                    break
+                        except:
```

Ruff (0.15.17)

[error] Do not use bare `except`

(E722)